src/auto-evo/AutoEvoExploringTool.tscn

```diff
@@ -762,80 +758,7 @@ margin_bottom = 486.0
 size_flags_horizontal = 3
 size_flags_vertical = 3
 
-[node name="SpeciesDetailPanel" type="PanelContainer" parent="GUI/VBoxContainer/HistoryReportSplit/Viewer/HBoxContainer"]
```

Comment:
Here you should preserve the PanelContainer, MarginContainer, VBoxContainer, MarginContainer and the Label.

Reply:
The FossilisationButton exists outside of the second MarginContainer, so I can't move that out of the SpeciesDetailsPanel (the entire point of creating it was to have common fossilisation logic). Preserving the other three seems to prevent the FossilisationDialog showing, again for reasons I don't have time to figure out.